src/components/pages/Books.js

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import './books.css';
+
+function Books() {
```

Comment:
- [ ] Please specify two different components, one for all the books list, and one component to render a single book